Resources/Prototypes/Entities/Objects/Devices/plant_analyzer.yml

```diff
@@ -0,0 +1,48 @@
+- type: entity
+  name: plant analyzer
+  parent: BaseItem
+  id: PlantAnalyzerUnpowered
```

Comment:
What is this unpowered version intended for? Debugging only?

Reply:
Yes, that was the intention, I saw the same with the healthanalyzer so I copied it.